STUDIO/org.openl.rules.repository.azure/test-resources/azureCommitOldStyle.yaml

```diff
@@ -0,0 +1,9 @@
+!!org.openl.rules.repository.azure.AzureCommit
```

Comment:
Can we omit this line?

Reply:
We cannot. Because It's an old yaml file that was created by previous SnakeYaml mapper. Added to check backward capability